deploy/kustomize/deploy_kustomize.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+
+set -x
+
+# Setup Environment Variables
+HCO_VERSION="${HCO_VERSION:-1.1.0}"
+HCO_CHANNEL="${HCO_CHANNEL:-1.1.0}"
+MARKETPLACE_MODE="${MARKETPLACE_MODE:-true}"
+HCO_REGISTRY_IMAGE="${HCO_REGISTRY_IMAGE:-quay.io/kubevirt/hco-container-registry:latest}"
+PRIVATE_REPO="${PRIVATE_REPO:-false}"
+QUAY_USERNAME="${QUAY_USERNAME:-}"
+QUAY_PASSWORD="${QUAY_PASSWORD:-}"
+CONTENT_ONLY="${CONTENT_ONLY:-false}"
+KVM_EMULATION="${KVM_EMULATION:-false}"
+OC_TOOL="${OC_TOOL:-oc}"
+
+#####################
+
+main() {
+  SCRIPT_DIR="$(dirname "$0")"
+  TARGET_NAMESPACE=$(grep name: $SCRIPT_DIR/namespace.yaml | awk '{print $2}')
```

Comment:
The other option is to take the`TARGET_NAMESPACE` argument (or environment variable) and simply write out the `namespace.yaml`.